### PR TITLE
feat(dedicated.vrack-services): dashboard information page without api

### PR DESCRIPTION
**Pull request by qpavy**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/vrack-services`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-11198
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- ~~[ ] Breaking change is mentioned in relevant commits~~

### CODE REVIEW

**Review by BrunoMarquesOVH — APPROVED** [review]

File: packages/manager/apps/container/src/container/legacy/server-sidebar/universe/DedicatedSidebar.tsx [hunk 1]
```diff
@@ -393,15 +393,6 @@ export default function DedicatedSidebar() {
...
 
-    if (feature.ip) {
-      menu.push({
-        id: 'dedicated-ip',
-        label: t('sidebar_ip_short'),
-        icon: getIcon('ovh-font ovh-font-ip'),
-        href: navigation.getURL('dedicated', '#/ip'),
-        routeMatcher: new RegExp('/ip(/|$)'),
-      });
-    }
```
Comment (anooparveti) [inline packages/manager/apps/container/src/container/legacy/server-sidebar/universe/DedicatedSidebar.tsx:404]: **question:** Why `IP` is removed in a PR concerning `vRack Services`?

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/apps/container/src/container/legacy/server-sidebar/universe/DedicatedSidebar.tsx (hunk 1 above)
Comment (qpavy) [reply packages/manager/apps/container/src/container/legacy/server-sidebar/universe/DedicatedSidebar.tsx:404]: It was a rebase mistake the ip was moved inside network by a ticket made by David few weeks ago

File: packages/manager/modules/vrack-services/src/routing.js [hunk 2]
```diff
@@ -20,14 +19,15 @@ export default /* @ngInject */ ($stateProvider) => {
           data,
         }));
       },
+      hideBreadcrumb: () => true,
+      breadcrumb: () => 'vRack Services',
```
[the remark this answers is not in the document]
Comment (qpavy) [reply packages/manager/modules/vrack-services/src/routing.js:23]: hideBreadcrumb hide the breadcrumb if you are on the current page but is showing for the child

File: packages/manager/modules/vrack-services/src/dashboard/template.html [hunk 3]
```diff
@@ -1,13 +1,85 @@
...
                 active="$ctrl.currentActiveLink() === $ctrl.dashboardLink"
             >
-                <span>Home</span>
+                <span data-translate="vrack_services_dashboard_tab_1"></span>
+            </oui-header-tabs-item>
+            <oui-header-tabs-item
+                href="{{:: $ctrl.subnetLink }}"
+                active="$ctrl.currentActiveLink() === $ctrl.subnetLink"
+            >
+                <span data-translate="vrack_services_dashboard_tab_2"></span>
```
Comment (anooparveti) [inline packages/manager/modules/vrack-services/src/dashboard/template.html:17]: **nitpick:** Replace `tab_1` with meaningful name
```suggestion
                <span data-translate="vrack_services_dashboard_subnet"></span>
```

File: packages/manager/modules/vrack-services/src/dashboard/template.html [hunk 4]
```diff
@@ -1,13 +1,85 @@
...
+            </oui-header-tabs-item>
+            <oui-header-tabs-item
+                href="{{:: $ctrl.endpointsLink }}"
+                active="$ctrl.currentActiveLink() === $ctrl.endpointsLink"
+            >
+                <span data-translate="vrack_services_dashboard_tab_3"></span>
```
Comment (anooparveti) [inline packages/manager/modules/vrack-services/src/dashboard/template.html:23]: **nitpick:** Replace `tab_1` with meaningful name
```suggestion
                <span data-translate="vrack_services_dashboard_endpoint"></span>
```

**Review by anooparveti — CHANGES_REQUESTED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**